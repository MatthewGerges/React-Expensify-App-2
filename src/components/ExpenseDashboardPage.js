import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
//connect connects your component to the redux store

const ExpenseDashboardPage = () => (
    <div>
    <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default ExpenseDashboardPage;