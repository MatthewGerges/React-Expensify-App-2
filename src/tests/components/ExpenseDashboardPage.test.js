import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';


test('should render ExpenseDashboardPage component correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot()
}) 