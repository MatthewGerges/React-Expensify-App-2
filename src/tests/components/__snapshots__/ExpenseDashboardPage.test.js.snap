// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage component correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
